net.nymtech.NymVPN.yml

```diff
@@ -0,0 +1,48 @@
+id: net.nymtech.NymVPN
+runtime: org.gnome.Platform
+runtime-version: '46'
+sdk: org.gnome.Sdk
+
+command: run.sh
+finish-args:
+  - --socket=wayland
+  - --socket=fallback-x11
+  - --device=dri
+  - --share=ipc
+  - --filesystem=/run/nym-vpn.sock:ro
+
+modules:
+  - shared-modules/libayatana-appindicator/libayatana-appindicator-gtk3.json
+  - name: nym-vpn
+    buildsystem: simple
+    sources:
+      - type: file
+        url: https://github.com/nymtech/nym-vpn-client/releases/download/nym-vpn-app-v0.2.3/nym-vpn_0.2.3_linux_x64
```

Comment:
why not building from source?

Reply:
As the build is not "trivial" as it needs to build 2 subprojects, a node and a rust project (protoc needed too), all that in a monorepo context.
I'd like to avoid it and use pre compiled binary from our GitHub release builds if it is possible. At least as a 1st iteration.